### ISSUE TITLE
add support for pulsar avro

### ISSUE THREAD

**Issue by NiuBlibing** [microsoft/pylance-release#4117]
## Environment data

-   Language Server version: Pylance  v2023.3.20
-   OS and version: Manjaro 5.15.94-1
-   Python version (& distribution if applicable, e.g. Anaconda): Python 3.10.9

## Code Snippet

```python
from pulsar.schema import Record, String, AvroSchema

class Test(Record):
    a = String()

schema = AvroSchema(Test)
# the following is shown as "Code is unreachable" by pylance
print(schema)
```

## Repro Steps

1. install depencies: `pip install 'pulsar-client[avro]==3.1.0'`

## Expected behavior

The code after the `AvroSchema` should be shown as reachable.

## Actual behavior

The code after the `AvroSchema` was shown as unreachable.

**Comment by erictraut** [comment]
This is addressed in the latest pyright code and will be addressed in a future release of pylance.

**Comment by debonte** [comment]
This issue has been fixed in prerelease version 2023.3.31, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#2023331-22-March-2023-PreRelease)
